pyspecdata/plot_funcs/pcolormesh.py

```diff
@@ -54,7 +53,7 @@ def pcolormesh(
             fig = plt.gcf()
             ax1 = fig.add_subplot(1, 1, 1)
         ax_list = [(ax1, lambda x: x.real, "real")]
-    X, Y = np.meshgrid(
+    X, Y = plt.meshgrid(
```

Comment:
I might have said incorrectly before, but this ultimately comes from numpy -- so not changing this is better.